src/index.html

```diff
@@ -1,12 +1,353 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="Moyo-logo">
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#!" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#!" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#!" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a
+              href="#!"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#!" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#!" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#!" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#!" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page__catalog catalog">
+      <div class="catalog__card card" data-qa="card">
+        <a href="#!" class="card__link">
+          <img
+            src="./images/imac.jpeg"
+            alt="Moyo-Product-imac/APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+        </a>
+        <a href="#!" class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+        <div class="card__product-code">Product code: 195434</div>
+        <a href="#!" class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__stars-review">Reviews: 5</span>
+        </a>
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/222697640-6f71e050-6438-4fb4-9433-a9efcad07565.png)
```suggestion
        <a href="#!" class="card__link">
          <img
            src="./images/imac.jpeg"
            alt="Moyo-Product-imac/APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
            class="card__img"
          >
        </a>
        
        <a href="#!" class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </a>
        
        <div class="card__product-code">Product code: 195434</div>
        
        <a href="#!" class="card__stars">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
          
          <span class="card__stars-review">Reviews: 5</span>
        </a>
```